controller.rst

```diff
@@ -262,4 +264,95 @@ Controller without subscription:
+
+1. Pay attention to warnings: If you see a version mismatch warning, consider updating your system when convenient.
+2. Keep your system updated: Regularly check for and apply updates to both your controller and firewall units to ensure the best compatibility and access to new features.
+3. Report issues: If you encounter any unusual behavior or errors, especially after seeing a version warning, report it to your IT support team or Nethsecurity support.
+
+Version awareness is a behind-the-scenes feature that helps keep your Nethsecurity system running smoothly. By automatically checking compatibility between the controller
+and units, it prevents many potential issues before they can affect your network. While it doesn't require any action from you, being aware of this feature can help you better understand and manage your system.
+
+Remember, if you're ever unsure about a version mismatch or update, it's always best to consult with your IT department or Nethsecurity support for guidance.
+
+.. rubric:: Bypass version awareness
+
+While version awareness is a useful feature, knowing the risks and potential issues, you may want to bypass it in some cases.
+To do so, the procedure is as follows:
+
+1. On the controller, go to the unit manager page and click on :guilabel:`More Info` of the unit you want to connect to.
+2. Copy the `Unit ID` value.
+3. Click on :guilabel:`Open SSH terminal`
+4. When the modal opens, you can safely close it. This was only needed to exchange some credentials with the unit.
+5. Open a new tab, and go to this URL: `https://<controller-fqdn>/#/controller/manage/<unit-id>/dashboard`. Example: `https://controller.nethsecurity.org/#/controller/manage/000000000-0000-0000-0000-000000000000/dashboard`.
+6. You will be able to access the unit's UI without the version check.
+
+.. rubric:: Update unit with SSH
+
+You can update the unit without connecting to it using the SSH terminal.
+Follow the steps to connect to the unit using :ref:`SSH Access <controller_ssh-section>`.
+
+Once connected, you can check for updates depending on what you want to update.
+
+a. Install package updates on the unit:
+
+   1. To check for updates for packages use the following command:
+ 
+      .. code-block:: bash
+ 
+        /usr/libexec/rpcd/ns.update call check-package-updates
+   
+   2. If you're ok with the installation of the packages you can run the following command:
+ 
+      .. code-block:: bash
+ 
+        /usr/libexec/rpcd/ns.update call install-package-updates
+
+b. To update the image, you can simply schedule the installation, remember this is a operation that restarts the firewall (causing a downtime)
+
+   1. Check if there is an updated image available:
+ 
+      .. code-block:: bash
+ 
+        /usr/libexec/rpcd/ns.update call check-system-update
+
+   2. If you want to proceed with the update, you must first generate a timestamp for the update, this can be done with the following command:
```

Comment:
Scheduling from command line is a bit tricky.
Why not documenting only the immediate upgrade procuder?

Reply:
I figured this would come up. And I agree :+1: